fvtest/porttest/omrstrTest.cpp

```diff
@@ -1542,7 +1552,7 @@ TEST(PortStrTest, str_test_atoe_vsnprintf)
 
 	/* min_width is less than the length of inputString */
 	portTestEnv->log("\n\tTesting case 2\n");
-	rc |= printToBuffer(OMRPORTLIB, FALSE, buffer2, bufferLength2, expectedOutput2, "%*s", 2, nonNullTerminatedString);
+	rc |= printToBuffer(OMRPORTLIB, FALSE, buffer2, bufferLength2, expectedOutput2, "%*s", 2, nullTerminatedString);
```

Comment:
Please don't change this test.

Reply:
This test doesn't seem correct. the code picks chars from the argument to either see a '\0' or the `precision` number of characters were picked. in this case no precision is specifies neither there is a termination char. the test pass with nonNullTerminated but I guess is just luck and because the value of buff[length] which is outside of the buff, is often 0. The `2` here is the min_width and has no impact on this behaviour.
it was okay before because the code would stop evaluation after `bufferLength2` items but it was not correct and is not the case anymore.